contracts/test-contract/schema/test-contract.json

```diff
@@ -0,0 +1,5645 @@
+{
+  "contract_name": "test-contract",
```

Comment:
Please delete this file since is not required

Reply:
Removed.